.rubocop.yml

```diff
@@ -16,10 +18,10 @@ Metrics/AbcSize:
 Metrics/ClassLength:
   Max: 150
 Metrics/BlockLength:
-  IgnoredMethods: ['describe']
+  AllowedMethods:
+    - 'describe'
   Max: 30
 
-
```

Comment:
- [ ] It would be nice if you make your pull request summary more detailed and kindly add the things that you have done in this pull request to make your pull request professional 👍
- [ ] Great work on having a readme file but kindly use the Microverse Readme template such as Headings and links etc 👍 to make your readme descriptive 😊